python/shotgun_model/shotgun_hierarchy_model.py

```diff
@@ -60,6 +70,8 @@ def __init__(self, parent, schema_generation=0, bg_task_manager=None):
         """
         super(ShotgunHierarchyModel, self).__init__(parent, bg_task_manager)
 
+        self._logger = sgtk.platform.get_logger(__name__)
```

Comment:
our recommended pattern is to keep a logger global at top of the file, not per class instance like this. I suggest moving it to the top unless you have a good reason not to (in that case, please add a comment!)